db/seeds.rb

```diff
@@ -1,7 +1,2 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+Role.create(name: "admin")
+Role.create(name: "curator")
```

Comment:
Instead of the string "curator", you might be able to use User::CURATOR_GROUP  if that class is accessible when the database is seeded with this (not sure it if is or not).